### ISSUE TITLE
add text to explain the time frame for the FTO authors section

### ISSUE THREAD

**Issue by govindgoel** [publiclab/community-toolbox#414]
There is a need to add text to explain the time frame for the FTO Authors section

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by govindgoel** [comment]
@akanksha-v  you can work on this

**Comment by akanksha-v** [comment]
Thank you @govindgoel  :)

**Comment by jywarren** [comment]
Ah, @govindgoel you noted in the chatroom:

> Here, the issue is that we cannot specify last 2 weeks as the api list the authors of fto till now. So we can add filter like the recent contributiors and change it to recent fto authors.

Does this just mean it lists for all time, right now? Why are there so few? Is it for all time but only open issues?

That's actually still quite useful. Maybe we need a way to view EITHER FTO authors from past time period (2 weeks?) AND alternatively for all currently open FTO issues? What do you both think, and have I understood properly? Thanks!!!

**Comment by govindgoel** [comment]
Yeah it is for all time but filters out open issues then list the authors, so we can add a filter for the recent and past authors.

**Comment by akanksha-v** [comment]
Yes @jywarren the list is for all-time FTO authors.

**Comment by jywarren** [comment]
Ok, do you think it's possible to fetch all from authors in a reusable
function, and have a parameter that lets us select open or closed issues?
Then we can start thinking about the design for how to display them!

I'm happy to help find this code or brainstorm the structure!

**Comment by govindgoel** [comment]
I will look at it now was occupied with academics, and update it here asap.

**Comment by YogeshSharma01** [comment]
Hi, @govindgoel  can I work on this?

**Comment by govindgoel** [comment]
@YogeshSharma01 I am probably working on it will push the changes asap.

**Comment by YogeshSharma01** [comment]
Yeah Sure

**Comment by jywarren** [comment]
Hi folks - i also opened a bigger issue around adjustable timeframe filters in #450 related to this. Thank you!!!
